modules/core/server-ts/app.ts

```diff
@@ -1,39 +1,30 @@
-import express from 'express';
+import express, { Express } from 'express';
 import path from 'path';
 import { GraphQLSchema } from 'graphql';
+import ServerModule, { Ware, CreateGraphQLContext } from '@gqlapp/module-server-ts';
 
-import { isApiExternal } from '@gqlapp/core-common';
-import ServerModule from '@gqlapp/module-server-ts';
-
-import graphiqlMiddleware from './middleware/graphiql';
-import websiteMiddleware from './middleware/website';
-import createApolloServer from './graphql';
 import errorMiddleware from './middleware/error';
 
+type TApplyWare = (ware: Ware) => void;
```

Comment:
Why `TApplyWare`, why do you add `T` here, did you see this practice anywhere in the kit?